doc/de/Quick-Start-guide.md

```diff
@@ -1,17 +1,24 @@
 Erste Schritte... 
-==========
+=================
 
 * [Zur Startseite der Hilfe](help)
 
-Das Erste zum Anfang: geh sicher, dass du schon eingeloggt bist. Wenn du noch nicht eingeloggt bist, kannst du das in dem Fenster unten machen. 
+Als Erstes: Gehe sicher, dass Du eingeloggt bist. Wenn Du noch nicht eingeloggt bist, kannst Du das in dem Fenster unten machen. 
 
-Sobald du eingeloggt bist (oder wenn du bereits eingeloggt bist), kannst du unten nun auf deine Profilseite schauen.
+Sobald dies geschehen ist, schaust Du auf Deine Profilseite (https://***************.tls/profile/*****).
 
-Hier sieht es ein wenig wie auf deiner Facebook-Seite aus. Hier findest du alle deine Statusmeldungen und Nachrichten deiner Freunde, die direkt auf deine Seite ("Wall") geschrieben haben. Um deinen Status einzutragen, klicke einfach auf die Box oben, in der "Teilen" steht. Wenn du das machst, vergrößert sich die Box. Nun kannst du einige Formatierungsoptionen wie Fett, kursiv, unterstrichen auswählen und ebenfalls Bilder und Links hinzufügen. Unten findest du in diesem Feld weitere Links, mit denen du Bilder und Dateien von deinem Computer hochladen, Webseiten mit einem Kurztext teilen und Video- und Audiodateien aus dem Internet einfügen kannst. Außerdem kannst du hier eintragen, wo du gerade bist. 
+Hier sieht es ein wenig wie auf (D)einer Facebook-Seite aus. Du findest hier alle Deine Statusmeldungen und Nachrichten Deiner Freunde, die direkt auf Deine "Pinnwand" ("Wall") geschrieben haben. Um Deinen Status einzutragen, klicke einfach auf die Box oben, in der "Teilen" steht. Wenn Du das machst, vergrößert sich die Box, und Du kannst nun Deinen Text eintragen, mit Hilfe einiger Formatierungsoptionen wie fett, kursiv, unterstrichen formatieren und ebenfalls Bilder und Links hinzufügen. Unten findest Du in diesem Feld weitere Knöpfe, mit denen Du Bilder und Dateien von Deinem Computer hochladen, Webseiten mit einem Kurztext teilen und Video- und Audiodateien aus dem Internet einfügen kannst. Außerdem kannst Du hier eintragen, wo Du gerade bist. 
 
-Wenn du deinen Beitrag ("Post") geschrieben hast, kannst du auf das "Schloss"-Symbol klicken und festlegen, wer deinen Beitrag sehen kann. Wenn du dieses Symbol nicht anklickst, ist dein Beitrag öffentlich. Das bedeutet, dass jeder, der dein Profil ansieht, der auf dem "Community"-Tab deines Servers oder auf dem "Netzwerk"-Tab ("Beiträge deiner Kontakte") eines befreundeten Kontakts ist, den Beitrag sehen kann.
+Wenn Du Deinen Beitrag ("Post") geschrieben hast, kannst Du auf das "Schloss"-Symbol klicken und festlegen, wer Deinen Beitrag sehen kann. Wenn Du dieses Symbol nicht anklickst, ist Dein Beitrag öffentlich. 
+Ein öffentlicher Beitrag ist sichbar für
+<ul>
+	<li>Besucher Deines Profils</li>
+	<li>Besucher Deiner "Gemeinschafts"-Seite</li>
```

Comment:
Das heißt, ich ergänze den Punkt "je nach Konfiguration des Knotens auch für alle Suchmaschinen aus dem übrigen Internet" !? 
Ich habe zum Beispiel bei meinem Knoten den Zugriff aus dem übrigen Internet unterbunden. Wenn ich nun aber einen öffentlichen Beitrag poste, dieser auf der Pinnwand eines Kontaktes landet, dessen Knoten nicht in der Art konfiguriert ist, so ist mein Beitrag dennoch für Google & Co sichtbar!?